### PR TITLE
test: add more e2e test scenarios for fluentd

### PR DESCRIPTION
**Pull request by ningziwen**
*Description of changes:*

- add more e2e test scenarios for fluentd
- make fluentd args doc format consistent

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: e2e/fluentd_test.go [hunk 1]
```diff
@@ -33,12 +40,80 @@ var testFluentd = func() {
...
+				containerIDKey:               testContainerID,
+				containerNameKey:             testContainerName,
+				fluentdSubSecondPrecisionKey: "false",
+			}
+			creator := cio.BinaryIO(*Binary, args)
+			err := sendTestLogByContainerd(creator, testLog)
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+			validateTestLogsInFluentd(fluentdLogDirName, testLog, testContainerID)
+		})
+		ginkgo.It("should send logs to fluentd log driver with sub-second-precision is false", func() {
```
Comment (ginglis13) [inline e2e/fluentd_test.go:87]: duplicate test?

File: e2e/main_test.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ const (
 	splunkDriverName                 = "splunk"
 	containerIDKey                   = "--container-id"
 	containerNameKey                 = "--container-name"
-	testContainerID                  = "test-container-id"
+	testContainerID                  = "210987654321"
```
Comment (ginglis13) [inline e2e/main_test.go:29]: Why the change from `test-container-id`?

**Review by ginglis13 — COMMENTED** [review]
some nits otherwise LGTM

File: e2e/main_test.go (hunk 2 above)
Comment (ningziwen) [reply e2e/main_test.go:29]: Fluentd only keeps the first 12 digits so it would be "test-contain" from the original one. We can't match them directly.

I think it is natural to make the testContainerID as real container ID format so didn't leave comment here.

Comment (ginglis13) [reply e2e/main_test.go:29]: cool thanks for the info!

**Review by ginglis13 — APPROVED** [review]

**Review by sam-berning — APPROVED** [review]
